charts/am-org-role-mapping-service/values.preview.template.yaml

```diff
@@ -29,7 +29,7 @@ java:
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     CASE_WORKER_REF_APP_URL: http://rd-caseworker-ref-api-aat.service.core-compute-aat.internal
-    JUDICIAL_REF_APP_URL: http://rd-judicial-api-aat.service.core-compute-aat.internal
+    JUDICIAL_REF_APP_URL: http://rd-judicial-api-pr-188.service.core-compute-preview.internal
```

Comment:
this should point to AAT now as FTA is disabled.

Reply:
roleback to AAT